photon-lib/py/photonlibpy/photonCamera.py

```diff
@@ -236,12 +236,13 @@ def _versionCheck(self) -> None:
 
         remoteUUID = self._rawBytesEntry.getTopic().getProperty("message_uuid")
 
-        if remoteUUID is None or len(remoteUUID) == 0:
+        if not remoteUUID:
```

Comment:
Needs to also check if renoteUUID is the string "null" and bail if so

```suggestion
        if not remoteUUID or remoteUUID == "null":
```

Reply:
Isn't that already covered by the condition below?

Reply:
No. If the remote uuid is "null", that means the coprocessor hasn't published to the topic yet and the property isn't set yet.

Reply:
Anyways; technically out of scope for this PR, but absolutely a bug, and needs to be fixed. seems to also be a bug in c++ as well, potentially:

https://github.com/PhotonVision/photonvision/blob/1fc93bd05dbdb79efc938a4aadf57d49067b4231/photon-targeting/src/main/java/org/photonvision/common/networktables/PacketSubscriber.java#L96